### ISSUE TITLE
Possible eclair/lnd interop issue related to fee updates

### ISSUE THREAD

**Issue by pm47** [lightningnetwork/lnd#1082]
### Background

lnd ---- eclair (channel initiated by lnd)

### Your environment

unknown (lnd is the remote node, I'm pretty sure it is fairly recent)

### Actual behaviour

Here is what happened (see [eclair logs](https://zerobin.net/?1fe9f2f6b6fc858d#iJJPVE5KaNDCCdacUPsixMnfhgLma7oSx2z+Ni8/Abw=), I'm trying to get lnd logs as well):

```
  lnd           eclair
   |                   |
   | (channel opening) |
   |                   |
   |------- fee1 ----->|
   |------- sig ------>|
   |<------ rev -------|
   |<------ sig -------|
   |------- rev ------>|
   |                   |
   |   (reconnection)  |
   |                   |
   |------- fee2 ----->|
   |------- sig ------>|
   |<------ rev -------|
   |<------ sig -------|
   |                   | <--- lnd doesn't send a rev back
   |                   |
   |------- fee2 ----->|
   |------- fee2 ----->|
   |------- fee2 ----->| <--- lnd keeps sending the same fee2 over and over again
   |        ...        |
```

**Comment by joaodealmeida** [comment]
lncli version 0.4.1 commit=13945de8060ad42e077988c1cf69e205dc11e528

[LND logs](https://zerobin.net/?c3c7c7a4fa55adc2#qYM9JXZuAUBpegYMBsZRV0ZsxXihDmkvLIZBvwI3O6c=)

**Comment by joaodealmeida** [comment]
Last time I restarted lnd was around 12h before that channel got closed. The backend has been stable since I started it (~15 days ago). I'm using bitcoind

I can dump here the lnd session log, but it's huge (it had the leveldebug set to info).

**Comment by pm47** [comment]
I got the same behavior with yalls node. Again right after a reconnect, `lnd` doesn't send a `rev` in response to a `sig`. I can provide full logs if useful.

@halseth I see you just added labels, is this a known issue?

**Comment by Roasbeef** [comment]
Labels are just to better categorize things in the issue tracker.

**Comment by Roasbeef** [comment]
Which y'alls node?

**Comment by pm47** [comment]
The new one: `03e50492eab4107a773141bb419e107bda3de3d55652e6e1a41225f06a0bbf2d56`

**Comment by pm47** [comment]
Resetting the connection didn't help:

```
  lnd           eclair
   |                   |
   |       ...         |
   |                   |
   |   (reconnection)  |
   |                   |
   |----- ann_sig ---->|
   |<---- ann_sig -----|
   |                   |
   |<------ fee1 ------|
   |<------ sig -------| <--- lnd doesn't send a rev back
   |                   |
   |<------ fee2 ------|
   |<------ fee3 ------|
   |<------ add -------| <--- eclair sends more fee/add updates
   |                   |
   |        ...        |
   |                   |
   |   (reconnection)  | <--- I reset the connection, all unsigned updates are dropped
   |                   |
   |<------ fee1 ------|
   |<------ sig -------| <--- lnd still doesn't send a rev back
   |                   |
   |        ...        |
```

**Comment by halseth** [comment]
Does it look like the `lnd` node is still connected/active after you send the updates? Or any indications that it crashed/disconnected?

**Comment by pm47** [comment]
Yes `lnd` is still connected and active, I see incoming data with `tcpdump`

**Comment by Roasbeef** [comment]
@pm47 what was the funding tx of the chan with y'alls? Have logs now from the operator.

**Comment by halseth** [comment]
@pm47 This something you have seen recently?

**Comment by pm47** [comment]
No, I haven't seen this in a while. Do you think issue may have been fixed as part of another change?

**Comment by halseth** [comment]
@pm47 Yeah, the fix for #1127 _miight_ have fixed it, as before it we wouldn't take fee updates into account when deciding if we should send a `CommitSig`. However, I'm not sure, as this report seems to indicate we are rapidly(?) sending fee updates, which should be unrelated.

**Comment by Roasbeef** [comment]
Closing for now as it hasn't been seen in the wild for some time. Will re-open if it arises. I think there's one other thing going on atm w.r.t eclair/lnd disconnecting and not being able to establish a channel. If that keeps popping up after our latest release, then will create a targeted issue for it.
